src/workerd/io/trace.c++

```diff
@@ -253,9 +253,10 @@ Trace::Exception::Exception(rpc::Trace::Exception::Reader reader)
       name(kj::str(reader.getName())),
       message(kj::str(reader.getMessage())) {}
 
-Tracer::Span::Span(kj::Own<Tracer> tracer, kj::Maybe<Jaeger::SpanData> spanData)
+Tracer::Span::Span(kj::Own<Tracer> tracer, kj::Maybe<Jaeger::SpanData> spanData,
+        kj::TimePoint durationStartTime)
     : tracer(kj::mv(tracer)), spanData(kj::mv(spanData)),
-      durationStartTime(kj::systemPreciseMonotonicClock().now()) {}
```

Comment:
I wonder why this doesn't need/have an `isPredictableModeForTest()`...

Reply:
Yeah, I wondered about that, too... and probably wrote the original code. :I  It's because the measurement is only used for calculating the duration, and the duration calculation itself is covered by `isPredictableModeForTest()`.

I guess it wouldn't hurt much to just use `isPredictableModeForTest()` everywhere, or define a common function that does that.  We already have several places that are defining something like `getTimeForJaegerSpan()` locally.

Reply:
Alternatively, a comment would probably suffice.